codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithygo/localservice/nameresolver/DafnyNameResolver.java

```diff
@@ -30,6 +30,13 @@ public static String dafnyNamespace(final Shape shape) {
                     .concat(INTERNAL_DAFNY);
     }
 
+    /**
```

Comment:
This came from auto merge and should be removed. I will do it once CI passes for this PR.